### PR TITLE
chore: no need to process check_suite not from mergify

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: Id7c349debd94a5f1efdc59c05479b310817c68a7

### CODE REVIEW

File: mergify_engine/github_events.py [hunk 1]
```diff
@@ -263,12 +263,11 @@ async def push_to_worker(
         if event["repository"]["archived"]:
             ignore_reason = "repository archived"
 
-        elif event["action"] != "rerequested":
-            ignore_reason = f"check_suite/{event['action']}"
+        elif event[event_type]["app"]["id"] != config.INTEGRATION_ID:
+            ignore_reason = "not a mergify check_suite"
```
Comment (jd) [inline mergify_engine/github_events.py:267]: I'm lacking context as to why we only process check suite from Mergify here?

Comment (sileht) [reply mergify_engine/github_events.py:267]: We use the check-suite/rerequested event to run the engine when people push the GitHub UI re-run button.

Comment (jd) [reply mergify_engine/github_events.py:267]: I'd thought we wanted to also be triggered on `check_suite` `completed` event just in case?

Comment (sileht) [reply mergify_engine/github_events.py:267]: Hum, ok, that looks legit seatbelt.

**Pull request closed without merging**